### DISCUSSION TITLE
My GitHub Stars Tab not working when logged in

### DISCUSSION THREAD

**Discussion by jgphilpott** [community/community#16246] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm having trouble viewing the [stars tab on my GitHub profile](https://github.com/jgphilpott?tab=stars) when logged in. When I visit the URL in an incognito window there is no problem and when I visit other peoples star tabs there is no problem. But when im logged in and I view my own stars tab I get a 502 error in the console and GitHub unicorn displayed in the view.

I have tried emptying the cache and refreshing and sometimes that works but then the problem comes back again! I have also tried in multiple browsers (Chrome, Firefox and Safari) ... same issue.

Any help or ideas about how I can stop this from happening would be much appreciated! Screenshot attached:

<img width="1440" alt="Screen Shot 2022-05-06 at 3 55 21 PM" src="https://user-images.githubusercontent.com/4128208/167406277-611995e7-8645-4081-bb2c-665e654c1136.png">

**Comment by queenofcorgis** [comment, marked as answer]
Hi there - are you still experiencing this and is anyone else experiencing this?

**Reply by jgphilpott** [reply]
Hey @liligalante, yes I am still experiencing this, it has been persistent for over a week now :(

The message says "This page is taking too long to load." but it can't be a connection speed issue because all the other tabs (Repos, Projects, Packages etc.) load very quickly and so do other sites I visit.

Sometimes it also returns a 500 internal server error but that's less frequent ... strange that the behaviour isn't consistent.

**Reply by FrederickHuangLin** [reply]
Hi @liligalante,

I have been encountering the same issue since a few weeks ago. The problem exists only when I log in to my account using the web browser. It works fine when I log out or switch to the app.

Thanks in advance if there are any suggestions.

**Reply by queenofcorgis** [reply]
@jgphilpott and @FrederickHuangLin thank you for letting me know - I've labeled this as a bug!

**Reply by gelcaas** [reply]
same problem

**Reply by jgphilpott** [reply]
**@liligalante thanks for taking action on this!** Seems to be fixed now, it's working fine... for me atleast. I've also noticed one other bug on the stars tab and opened another discussion about it.. take a look [here](https://github.com/github-community/community/discussions/19023) if your interested.

**Reply by FrederickHuangLin** [reply]
@liligalante I have no problem opening the stars tab as well. Thanks for your help!

**Reply by Nawarajkarki** [reply]
I am also having problem with the stars tab in my Github Profile. 
Before I had 3 Repositories stared but the stars tab showed 30 repo's stared. So I removed all the stared repositories and it shows 27 stared repositories but there are none.

**Reply by thimslugga** [reply]
I'm running into this issue myself. I noticed it only happens when accessing via website. I do not run into the issue when trying to access stars/lists via mobile iOS app.
